.github/workflows/php.yaml

```diff
@@ -0,0 +1,107 @@
+name: PHP
```

Comment:
New name! 🎉

May I add a spellcheck workflow? #56

Reply:
OK, lets try it :)

Reply:
Okay. In the next PR.